source/include/TypeAndDataManager.h

```diff
@@ -1329,13 +1328,17 @@ struct StringRefCmp {
         Int32 cmp = memcmp(a->Begin(), b->Begin(), Min(a->Length(), b->Length()));
         if (cmp < 0) {
             return true;
-        } else if (cmp > 0) {
+        }
```

Comment:
Please undo the changes to this function.  Leave the 'else' statements.
This is a good example of a situation where the code reads much better with them, regardless of whether they are 'redundant'.